# Translations template for cinder.
# Copyright (C) 2015 ORGANIZATION
# This file is distributed under the same license as the cinder project.
#
# Translators:
# OpenStack Infra <zanata@openstack.org>, 2015. #zanata
# Zbyněk Schwarz <zbynek.schwarz@gmail.com>, 2015. #zanata
# Andreas Jaeger <jaegerandi@gmail.com>, 2016. #zanata
msgid ""
msgstr ""
"Project-Id-Version: cinder 9.0.0.0b4.dev54\n"
"Report-Msgid-Bugs-To: https://bugs.launchpad.net/openstack-i18n/\n"
"POT-Creation-Date: 2016-09-06 23:17+0000\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"PO-Revision-Date: 2015-12-22 01:37+0000\n"
"Last-Translator: Zbyněk Schwarz <zbynek.schwarz@gmail.com>\n"
"Language: cs\n"
"Plural-Forms: nplurals=3; plural=(n==1) ? 0 : (n>=2 && n<=4) ? 1 : 2;\n"
"Generated-By: Babel 2.0\n"
"X-Generator: Zanata 3.7.3\n"
"Language-Team: Czech\n"

#, python-format
msgid "%(path)s is being set with open permissions: %(perm)s"
msgstr "%(path)s je nastavována s volnými oprávněními: %(perm)s"

#, python-format
msgid "%s"
msgstr "%s"

#, python-format
msgid "%s is already mounted"
msgstr "%s již je připojeno"

#, python-format
msgid "%s not found!"
msgstr "%s nenalezeno!"

msgid ""
"'hpe3par:cpg' is not supported as an extra spec in a volume type.  CPG's are "
"chosen by the cinder scheduler, as a pool, from the cinder.conf entry "
"'hpe3par_cpg', which can be a list of CPGs."
msgstr ""
"'hpe3par:cpg' není podporováno jako dodatečná specifikace v typu svazku. "
"Společné skupiny poskytování jsou voleny plánovačem cinder, jako zásoba, z "
"položky 'hpe3par_cpg' v cinder.conf, který může být seznamem těchto skupin."

#, python-format
msgid "3PAR vlun for volume %(name)s not found on host %(host)s"
msgstr "3PAR vlun pro svazek %(name)s nenalezen v hostiteli %(host)s"

msgid "Attempted to delete a space that's not there."
msgstr "Pokus smazat prostor který neexistuje."

#, python-format
msgid ""
"Attempting a rollback of: %(volumeName)s to original pool "
"%(sourcePoolInstanceName)s."
msgstr ""
"Pokus o zpětné vrácení %(volumeName)s do původní zásoby "
"%(sourcePoolInstanceName)s."

msgid "Attempting recreate of backing lun..."
msgstr "Pokus o znovuvytvoření záložního lun..."

#, python-format
msgid ""
"Availability zone '%(s_az)s' not found, falling back to '%(s_fallback_az)s'."
msgstr ""
"Zóna dostupnosti '%(s_az)s' nebyla nalezena, bude použita záložní "
"'%(s_fallback_az)s'."

#, python-format
msgid "Availability zone '%s' is invalid"
msgstr "Zóna dostupnosti '%s' je neplatná"

#, python-format
msgid ""
"Backup service %(service)s does not support verify. Backup id %(id)s is not "
"verified. Skipping verify."
msgstr ""
"Zálohovací služba %(service)s nepodporuje ověřování. Záloha s id %(id)s není "
"ověřena. Ověřování přeskočeno."

#, python-format
msgid "CHAP is enabled, but server secret not configured on server %s"
msgstr "CHAP je povolen, ale soukromý klíč serveru není nastaven na serveru %s"

#, python-format
msgid "CHAP secret exists for host %s but CHAP is disabled"
msgstr "Soukromý klíč CHAP pro hostitele %s existuje, ale CHAP je zakázán."

msgid "CHAP secret exists for host but CHAP is disabled."
msgstr "Soukromý klíč CHAP pro hostitele existuje, ale CHAP je zakázán."

msgid "Can't find lun on the array."
msgstr "Nelze najít lun v poli."

msgid "Can't find snapshot on the array."
msgstr "Nelze najít snímek v poli."

msgid "Can't find target iqn from rest."
msgstr "Nelze najít cílové iqn z REST."

msgid "Cannot determine the hardware type."
msgstr "Nelze zjistit typ hardwaru."

#, python-format
msgid "Cannot get volume status %(exc)s."
msgstr "Nelze získat stav svazku %(exc)s."

#, python-format
msgid ""
"Cannot undo volume rename; old name was %(old_name)s and new name is "
"%(new_name)s."
msgstr ""
"Nelze vrátit přejmenování svazku zpět; starý název byl %(old_name)s a nový "
"název je %(new_name)s."

#, python-format
msgid "Change will make usage less than 0 for the following resources: %s"
msgstr "Změna využití se sníží na méně než 0 pro následující zdroje: %s"

#, python-format
msgid ""
"Changing backing: %(backing)s name from %(new_name)s to %(old_name)s failed."
msgstr "Změna názvu zálohy %(backing)s z %(new_name)s na %(old_name)s selhala."

#, python-format
msgid ""
"Clone failed on V3. Cleaning up the target volume. Clone name: %(cloneName)s "
msgstr ""
"Klonování selhalo ve V3. Čištění cílového svazku. Název klona: %(cloneName)s "

msgid ""
"Configuration option eqlx_cli_timeout is deprecated and will be removed in M "
"release. Use ssh_conn_timeout instead."
msgstr ""
"Volba nastavení eqlx_cli_timeout je zastaralá a bude odstraněna ve verzi M. "
"Místo toho používejte ssh_conn_timeout."

msgid ""
"Configuration options eqlx_use_chap, eqlx_chap_login and eqlx_chap_password "
"are deprecated. Use use_chap_auth, chap_username and chap_password "
"respectively for the same."
msgstr ""
"Volby nastavení eqlx_use_chap, eqlx_chap_login a eqlx_chap_password jsou "
"zastaralé. Byly nahrazeny volbami use_chap_auth, chap_username a "
"chap_password respectively se stejnou funkcí."

#, python-format
msgid "Could not create target because it already exists for volume: %s"
msgstr "Nelze vytvořit cíl protože již ve svazku existuje: %s"

#, python-format
msgid "Could not determine root volume name on %s."
msgstr "Nelze zjistit název kořenového svazku v %s."

#, python-format
msgid "Could not get pool information (%s)!"
msgstr "Nelze získat informace o zásobě (%s)!"

#, python-format
msgid "Could not get status for %(name)s (%(status)d)."
msgstr "Nelze získat stav %(name)s (%(status)d)."

#, python-format
msgid ""
"CreateStorageHardwareID failed. initiator: %(initiator)s, rc=%(rc)d, ret="
"%(ret)s."
msgstr ""
"Vytvoření ID hardwaru úložiště selhalo. Zavaděč %(initiator)s, rc=%(rc)d, "
"ret=%(ret)s."

#, python-format
msgid "DELETE call failed for %s!"
msgstr "Volání smazání selhalo u %s!"

#, python-format
msgid "Deadlock detected when running '%(func_name)s': Retrying..."
msgstr ""
"Při provádění '%(func_name)s' zjištěno zablokování. Bude proveden nový "
"pokus..."

#, python-format
msgid "Delete Snapshot id not found. Removing from cinder: %(id)s Ex: %(msg)s"
msgstr ""
"ID mazaného snímku nenalezeno. Odstraňování z cinder: %(id)s Výjimka: %(msg)s"

#, python-format
msgid "Delete volume id not found. Removing from cinder: %(id)s Ex: %(msg)s"
msgstr ""
"ID mazanéhosvazku nenalezeno. Odstraňování z cinder: %(id)s Výjimka: %(msg)s"

#, python-format
msgid "Deleting image in unexpected status: %(image_status)s."
msgstr "Mazání obrazu v neočekávaném stavu: %(image_status)s."

#, python-format
msgid "Destination %s already exists."
msgstr "Cíl %s již existuje."

msgid ""
"Destination volume type is different from source volume type for an "
"encrypted volume. Encrypted backup restore has failed."
msgstr ""
"Cílový typ svazku se liší od zdrojového typu zašifrovaného svazku. Obnovení "
"zašifrované zálohy selhalo."

msgid "Detected snapshot stuck in creating status, setting to ERROR."
msgstr "Objeven snímek zaseknutý ve stavu creating, nastavování na ERROR."

#, python-format
msgid "Detected volume stuck in %(curr_status)s status, setting to ERROR."
msgstr ""
"Objeven svazek zaseknutý ve stavu %(curr_status)s, nastavování na ERROR."

msgid "Discover file retries exhausted."
msgstr "Vyčerpány pokusy o zjištění souboru."

msgid "Driver didn't return connection info from terminate_connection call."
msgstr "Ovladač nevrátil informace o připojení z volání o ukončení připojení."

msgid "Driver didn't return connection info, can't add zone."
msgstr "Ovladač nevrátil informace o připojení, nelze přidat zónu."

#, python-format
msgid ""
"Driver path %s is deprecated, update your configuration to the new path."
msgstr ""
"Cesta ovladače %s je zastaralá, aktualizujte svá nastavení na novou cestu."

#, python-format
msgid "Error encountered translating config_string: %(config_string)s to dict"
msgstr ""
"Při překladu řetězce nastavení se objevila chyba: %(config_string)s do dict"

#, python-format
msgid "Error finding LUNs for volume %s. Verify volume exists."
msgstr "Chyba při hledání LUN pro svazek %s. Ověřování zda svazek existuje."

#, python-format
msgid ""
"Error in filtering function '%(function)s' : '%(error)s' :: failing host"
msgstr ""
"Chyba ve funkce filtrování '%(function)s' : '%(error)s' :: hostitel selhává"

#, python-format
msgid ""
"Error in goodness_function function '%(function)s' : '%(error)s' :: "
"Defaulting to a goodness of 0"
msgstr ""
"Chyba ve funkcí Goodness '%(function)s' : '%(error)s' :: Je použita její "
"minimální hodnota 0"

#, python-format
msgid "Error mapping LUN. Code :%(code)s, Message: %(message)s"
msgstr "Chyba při mapování LUN. Kód: %(code)s, zpráva: %(message)s"

#, python-format
msgid "Error occurred while deleting backing: %s."
msgstr "Při mazání zálohy nastala chyba: %s."

#, python-format
msgid "Error occurred while deleting descriptor: %s."
msgstr "Při mazání popisovače nastala chyba: %s."

#, python-format
msgid "Error occurred while deleting temporary disk: %s."
msgstr "Při mazání dočasného disku nastala chyba: %s."

#, python-format
msgid "Error refreshing volume info. Message: %s"
msgstr "Chyba při obnovování informaci o svazku. Zpráva: %s"

#, python-format
msgid "Error running SSH command: %s"
msgstr "Chyba při provádění příkazu SSH: %s"

#, python-format
msgid "Error unmapping LUN. Code :%(code)s, Message: %(message)s"
msgstr "Chyba při rušení mapování LUN. Kód: %(code)s, zpráva: %(message)s"

#, python-format
msgid "Error updating agent-type for volume %s."
msgstr "Chyba při aktualizaci typu agentu ve svazku %s."

msgid "Error while listing objects, continuing with delete."
msgstr "Chyba při vypisování objektů, pokračuje se ve smazání."

#, python-format
msgid "Exception during cache cleaning %(share)s. Message - %(ex)s"
msgstr "Výjimka při čištění mezipaměti %(share)s. Zpráva - %(ex)s"

#, python-format
msgid "Exception during deleting %s"
msgstr "Během mazání %s nastala výjimka"

#, python-format
msgid "Exception during mounting %s"
msgstr "Při připojování %s došlo k výjimce"

#, python-format
msgid "Exception during unmounting %s"
msgstr "Při odpojování nastala výjimka %s"

#, python-format
msgid "Exception moving file %(src)s. Message - %(e)s"
msgstr "Výjimka při přesunování souboru %(src)s. Zpráva - %(e)s"

#, python-format
msgid "Exception moving file %(src)s. Message: %(e)s"
msgstr "Výjimka při přesunování souboru %(src)s. Zpráva: %(e)s"

#, python-format
msgid ""
"Exception while creating image %(image_id)s snapshot. Exception: %(exc)s"
msgstr "Při vytváření snímku obrazu %(image_id)s nastala výjimka: %(exc)s"

#, python-format
msgid ""
"Exception while registering image %(image_id)s in cache. Exception: %(exc)s"
msgstr "Výjimka při registraci obrazu %(image_id)s v mezipaměti: %(exc)s"

#, python-format
msgid ""
"Extension %(ext_name)s: Cannot extend resource %(collection)s: No such "
"resource"
msgstr ""
"Rozšíření %(ext_name)s: nelze rozšířit zdroj %(collection)s: Žádný takový "
"zdroj"

#, python-format
msgid "Extra spec %(old)s is deprecated.  Use %(new)s instead."
msgstr ""
"Dodatečná specifikace %(old)s je zastaralá. Místo toho použijte %(new)s."

#, python-format
msgid "Extra spec %(old)s is obsolete.  Use %(new)s instead."
msgstr "Dodatečná specifikace %(old)s je zrušena. Místo toho použijte %(new)s."

msgid ""
"Extra spec key 'storagetype:pool' is obsoleted since driver version 5.1.0. "
"This key will be ignored."
msgstr ""
"Klíč dodatečné specifikace 'storagetype:pool' je zastaralý od verze ovladače "
"5.1.0. Tento klíč bude ignorován."

#, python-format
msgid "FAST is enabled. Policy: %(fastPolicyName)s."
msgstr "FAST je povoleno. Zásada: %(fastPolicyName)s."

#, python-format
msgid ""
"Failed target removal because target or ACL's couldn't be found for iqn: %s."
msgstr ""
"Odstranění cíle selhalo protože cíl nebo ACL nemohly být v iqn nalezeny: %s."

#, python-format
msgid ""
"Failed terminating the connection of volume %(volume_id)s, but it is "
"acceptable."
msgstr ""
"Ukončení spojení se svazkem %(volume_id)s selhalo, selhání je přijatelné."

#, python-format
msgid "Failed to activate volume copy throttling: %(err)s"
msgstr "Nelze aktivovat přiškrcení kopie svazku: %(err)s"

#, python-format
msgid "Failed to add host group: %s"
msgstr "Nelze přidat skupinu hostitele: %s"

#, python-format
msgid "Failed to associate qos specs %(id)s with type: %(vol_type_id)s"
msgstr "Přidružení specifikace qos %(id)s k typu: %(vol_type_id)s selhalo"

#, python-format
msgid "Failed to create new image-volume cache entry. Error: %(exception)s"
msgstr ""
"Nelze vytvořit novou položku obrazu-svazku v mezipaměti. Výjimka: "
"%(exception)s"

#, python-format
msgid "Failed to create pair: %s"
msgstr "Nelze vytvořit pár: %s"

#, python-format
msgid ""
"Failed to create volume from image-volume cache, will fall back to default "
"behavior. Error: %(exception)s"
msgstr ""
"Nelze vytvořit svazek z mezipaměti obrazu-svazku, bude použito výchozí "
"chování. Chyba: %(exception)s"

#, python-format
msgid "Failed to destroy Storage Group %s."
msgstr "Nelze zničit skupinu úložiště %s."

#, python-format
msgid "Failed to disassociate qos specs %(id)s with type: %(vol_type_id)s"
msgstr "Odloučení specifikace qos %(id)s od typu: %(vol_type_id)s selhalo"

#, python-format
msgid "Failed to disassociate qos specs %s."
msgstr "Odloučení specifikace qos %s selhalo."

#, python-format
msgid "Failed to discard zero page: %s"
msgstr "Nelze zahodit nulovou stránku: %s"

msgid "Failed to get Raid Snapshot ID and did not store in snapshot."
msgstr "Nelze získat ID Raid snímku a ve snímku nebylo uloženo."

msgid "Failed to get target pool id."
msgstr "Nelze získat id cílové zásoby."

#, python-format
msgid "Failed to invoke ems. Message : %s"
msgstr "Nelze zavolat ems. Zpráva: %s"

#, python-format
msgid "Failed to load extension %(classpath)s: %(exc)s"
msgstr "Nelze načíst rozšíření %(classpath)s: %(exc)s"

#, python-format
msgid "Failed to load extension %(ext_factory)s: %(exc)s"
msgstr "Nelze načít rozšíření %(ext_factory)s: %(exc)s"

#, python-format
msgid "Failed to load extension %(ext_name)s: %(exc)s"
msgstr "Nelze načíst rozšíření %(ext_name)s: %(exc)s"

#, python-format
msgid "Failed to manage virtual volume %(disp)s due to error during retype."
msgstr ""
"Nelze spravovat virtuální svazek %(disp)s kvůli chybě během přetypování."

#, python-format
msgid ""
"Failed to migrate volume. The destination volume %(vol)s is not deleted "
"since the source volume may have been deleted."
msgstr ""
"Nelze přesunout svazek. Cílový svazek %(vol)s nebyl smazán protože zdrojový "
"svazek mohl být smazán."

#, python-format
msgid ""
"Failed to migrate: %(volumeName)s from default source storage group for FAST "
"policy: %(sourceFastPolicyName)s. Attempting cleanup... "
msgstr ""
"Nelze přesunout: %(volumeName)s z výchozí skupiny zdrojového úložiště pro "
"zásadu FAST  %(sourceFastPolicyName)s. Pokus o vyčištění..."

#, python-format
msgid "Failed to query pool %(id)s status %(ret)d."
msgstr "Dotaz na stav %(ret)d zásoby %(id)s selhal."

#, python-format
msgid "Failed to refresh mounts, reason=%s"
msgstr "Nelze obnovit připojení, důvod=%s"

#, python-format
msgid "Failed to restart horcm: %s"
msgstr "Nelze restartovat horcm: %s"

#, python-format
msgid "Failed to run command: %s."
msgstr "Nelze provést příkaz: %s."

#, python-format
msgid "Failed to run lsguicapability. Exception: %s."
msgstr "Nelze spustit lsguicapability. Výjimka: %s."

#, python-format
msgid ""
"Failed to save iscsi LIO configuration when modifying volume id: %(vol_id)s."
msgstr "Nelze uložit nastavení iscsi LIO při měnění id svazku %(vol_id)s."

#, python-format
msgid "Failed to setup blkio cgroup to throttle the device '%(device)s'."
msgstr ""
"Nelze nastavit kontrolní skupinu vstupu/výstupu bloku pro škrcení zařízení "
"'%(device)s'."

#, python-format
msgid "Failed to unmount previous mount: %s"
msgstr "Nelze odpojit předchozí připojení: %s"

#, python-format
msgid ""
"Failed to update %(conf)s for volume id %(vol_id)s after removing iscsi "
"target. %(conf)s does not exist."
msgstr ""
"Nelze aktualizovat %(conf)s pro svazek s id %(vol_id)s po odstranění cíle "
"iscsi. %(conf)s neexistuje."

#, python-format
msgid "Failure deleting job %s."
msgstr "Nelze smazat úkol %s."

#, python-format
msgid "Failure deleting temp snapshot %s."
msgstr "Mazání dočasného snímku %s selhalo."

#, python-format
msgid "Failure deleting the snapshot %(snapshot_id)s of volume %(volume_id)s."
msgstr "Mazání snímku %(snapshot_id)s svazku %(volume_id)s selhalo."

msgid ""
"Fallocate not supported by current version of glusterfs. So falling back to "
"dd."
msgstr ""
"Fallocate není podporováno současnou verzí glusterfs. Přechází se zpět na dd."

#, python-format
msgid ""
"Flexvisor failed to delete volume %(id)s from group %(vgid)s due to "
"%(status)s."
msgstr ""
"Flexvisor nemohl smazat svazek %(id)s ze skupiny %(vgid)s z důvodu%(status)s."

#, python-format
msgid "Flexvisor failed to delete volume %(id)s from the group %(vgid)s."
msgstr "Flexvisor nemohl smazat svazek %(id)s ze skupiny %(vgid)s."

#, python-format
msgid ""
"Found invalid iSCSI IP address(s) in configuration option(s) "
"hpe3par_iscsi_ips or iscsi_ip_address '%s.'"
msgstr ""
"Ve volbách nastavení hpe3par_iscsi_ips nebo iscsi_ip_address nalezeny "
"neplatné IP adresy iSCSI: '%s'."

msgid "Goodness function not set :: defaulting to minimal goodness rating of 0"
msgstr "Funkce Goodness není nastavena :: je použita její minimální hodnota 0"

#, python-format
msgid "Got disconnected; trying to reconnect. (%s)"
msgstr "Došlo k odpojení; pokus o znovu připojení. (%s)"

#, python-format
msgid ""
"Group sync name not found for target group %(target)s on %(storageSystem)s."
msgstr ""
"Název synchronizace skupiny nenalezen v cílové skupině %(target)s na "
"%(storageSystem)s."

#, python-format
msgid ""
"HPELeftHand API is version %(current)s. A minimum version of %(min)s is "
"needed for manage/unmanage support."
msgstr ""
"Verze API HPELeftHand: %(current)s. Pro podporu vytváření/rušení spravování "
"je vyžadována verze %(min)s nebo vyšší."

#, python-format
msgid ""
"Hint \"%s\" dropped because ExtendedServerAttributes not active in Nova."
msgstr ""
"Nápověda \"%s\" zahozena, protože rozšířené vlastnosti serveru nejsou "
"aktivní v Nova."

#, python-format
msgid ""
"Hint \"%s\" dropped because Nova did not return enough information. Either "
"Nova policy needs to be changed or a privileged account for Nova should be "
"specified in conf."
msgstr ""
"Nápověda \"%s\" zahozena, protože Nova nevrátila dostatek informací. Buď je "
"třeba změnit zásadu Nova, nebo je třeba v nastavení zadat výsadní účet pro "
"Nova se všemi potřebnými oprávněními."

msgid ""
"Host exists without CHAP credentials set and has iSCSI attachments but CHAP "
"is enabled. Updating host with new CHAP credentials."
msgstr ""
"Hostitel je bez nastavených přihlašovacích údajů CHAP, má připojení iSCSI, "
"ale CHAP je povolen. Aktualizování hostitele pomocí nových přihlašovacích "
"údajů CHAP."

msgid "Host has no CHAP key, but CHAP is enabled."
msgstr "Hostiteli nemá žádný klíč CHAP, ale CHAP je povolen."

#, python-format
msgid "IOnetworks GET failed (%d)"
msgstr "Získání IOnetworks selhalo (%d)"

msgid "IQN already existed."
msgstr "IQN již existuje."

msgid "IQN has been used to create map."
msgstr "IQN bylo použito k vytvoření mapy."

msgid "ISCSI provider_location not stored, using discovery"
msgstr "Umístění poskytovatele ISCSI neuloženo, bude se zjišťovat"

msgid "Id not in sort_keys; is sort_keys unique?"
msgstr "id není v seřazení klíčů; je seřazení jedinečné?"

msgid "Image delete encountered an error."
msgstr "Při mazání obrazu nastala chyba."

#, python-format
msgid "Image-volume cache for host %(host)s does not have enough space (GB)."
msgstr ""
"Mezipaměť obrazu-svazku pro hostitele %(host)s nemá dostatek místa (GB)."

msgid "Inconsistent Luns."
msgstr "Nekonzistentní Lun."

#, python-format
msgid ""
"Incorrect value error: %(blocksize)s, it may indicate that "
"'volume_dd_blocksize' was configured incorrectly. Fall back to default."
msgstr ""
"Chyba kvůli nesprávné hodnotě: %(blocksize)s, to může naznačovat, že "
"'volume_dd_blocksize' nebylo nastaveno správně. Bude použita výchozí hodnota."

#, python-format
msgid ""
"Insufficient free space for thin provisioning. The ratio of provisioned "
"capacity over total capacity %(provisioned_ratio).2f has exceeded the "
"maximum over subscription ratio %(oversub_ratio).2f on host %(host)s."
msgstr ""
"Pro mělké poskytování není dostatek volného místa. Poměr poskytované "
"kapacity na celkovou %(provisioned_ratio).2f překročil maximální poměr "
"odběru %(oversub_ratio).2f na hostiteli %(host)s."

#, python-format
msgid ""
"Insufficient free space for volume creation on host %(host)s (requested / "
"avail): %(requested)s/%(available)s"
msgstr ""
"Není dostatek volného místa pro vytvoření svazku na hostiteli %(host)s. "
"(Požadováno: %(requested)s/Dostupné %(available)s)"

#, python-format
msgid ""
"Insufficient free space for volume creation. Total capacity is %(total).2f "
"on host %(host)s."
msgstr ""
"Není dostatek volného místa pro vytvoření svazku. Celková kapacita je "
"%(total).2f na hostiteli %(host)s."

#, python-format
msgid "Invalid IP address format '%s'"
msgstr "Neplatný formát IP adresy '%s'"

#, python-format
msgid ""
"Invalid goodness result.  Result must be between 0 and 100.  Result "
"generated: '%s' :: Defaulting to a goodness of 0"
msgstr ""
"Neplatný výsledek funkce goodness. Výsledek musí být mezi 0 až 100. "
"Vypočtený výsledek '%s' :: Je použita její minimální hodnota 0"

#, python-format
msgid "Invalid trace flag: %s"
msgstr "Neplatný příznak sledování: %s"

msgid ""
"It is not the recommended way to use drivers by NetApp. Please use "
"NetAppDriver to achieve the functionality."
msgstr ""
"Toto není doporučený způsob používání ovladačů od NetApp. Pro dosažení této "
"funkce prosím použijte NetAppDriver."

#, python-format
msgid ""
"LUN misalignment may occur for current initiator group %(ig_nm)s) with host "
"OS type %(ig_os)s. Please configure initiator group manually according to "
"the type of the host OS."
msgstr ""
"U současné skupině zavaděče %(ig_nm)s s hostitelem mající typ OS %(ig_os)s "
"se může objevit nevyrovnanost LUN. Prosím nastavte skupinu zavaděče ručně "
"podle typu OS na hostiteli."

msgid "Least busy iSCSI port not found, using first iSCSI port in list."
msgstr ""
"Nejméně zaneprázdněný port iSCSI nenalezen, použit první port v seznamu."

#, python-format
msgid ""
"Lun is not in lungroup. Lun id: %(lun_id)s, lungroup id: %(lungroup_id)s"
msgstr ""
"Lun není ve skupine lun. ID LUN %(lun_id)s, ID skupiny LUN: %(lungroup_id)s."

#, python-format
msgid "Message - %s."
msgstr "Zpráva - %s."

#, python-format
msgid "More than one valid preset was detected, using %s"
msgstr "Byly zjištěny více než jedny platné předvolby, jsou použity %s"

#, python-format
msgid "Mount point %(name)s already exists. Message: %(msg)s"
msgstr "Bod připojení %(name)s již existuje. Zpráva %(msg)s"

msgid "No VLUN contained CHAP credentials. Generating new CHAP key."
msgstr ""
"Žádný VLUN neobsahoval přihlašovací údaje CHAP. Vytváření nového klíče CHAP."

#, python-format
msgid "No backing file found for %s, allowing snapshot to be deleted."
msgstr "Pro %s nenalezen žádný zálohovací soubor, smazání snímku povoleno."

#, python-format
msgid "No entry in LUN table for volume/snapshot %(name)s."
msgstr "Žádná položka v tablce LUN pro svazek/snímek %(name)s."

msgid "No host or VLUNs exist. Generating new CHAP key."
msgstr "Žádný VLUN nebo hostitel neexistuje. Vytváření nového klíče CHAP."

msgid "No mapping."
msgstr "Žádné mapování."

#, python-format
msgid "No port group found in masking view %(mv)s."
msgstr "V maskování %(mv)s nenalezena žádná skupiny portů."

msgid "No protection domain name or id was specified in configuration."
msgstr "V nastavení nebylo zadán žádný název nebo id ochranné domény."

#, python-format
msgid "No status payload for volume %s."
msgstr "Žádný obsah stavu svazku %s."

#, python-format
msgid ""
"No storage group found. Performing rollback on Volume: %(volumeName)s To "
"return it to the default storage group for FAST policy %(fastPolicyName)s."
msgstr ""
"Nenalezena žádná skupina úložiště. Provádění zpětného vrácení svazku "
"%(volumeName)s pro návrat do výchozí skupiny úložiště pro zásadu FAST "
"%(fastPolicyName)s."

#, python-format
msgid "No storage pool found with available capacity %s."
msgstr "Nenalezena žádná zásoba úložiště s dostupnou vlastností %s."

msgid "No storage pool name or id was found."
msgstr "Nebyla nalezen žádný název zásoby úložiště nebo jeho id."

msgid "No such host alias name."
msgstr "Žádný takový název zkratky hostitele."

#, python-format
msgid "No target ports found in masking view %(maskingView)s."
msgstr "V maskování %(maskingView)s nenalezeny žádné cílové porty."

msgid "No volume node in XML content."
msgstr "V obsahu XML není žádný uzel svazku."

#, python-format
msgid "No weighed hosts found for volume with properties: %s"
msgstr "Nenalezeni žádní vážení hostitelé pro svazek s vlastnostmi %s"

msgid "Non-iSCSI VLUN detected."
msgstr "Zjištěn VLUN  mimo iSCSI."

#, python-format
msgid "Not deleting key %s"
msgstr "Klíč %s nebude smazán"

#, python-format
msgid "Persistence file already exists for volume, found file at: %s"
msgstr "Soubor přetrvávání svazku již existuje, nalezen v %s"

#, python-format
msgid ""
"Pre check for deletion. Volume: %(volumeName)s is part of a storage group. "
"Attempting removal from %(storageGroupInstanceNames)s."
msgstr ""
"Předkontrola mazání. Svazek %(volumeName)s je součástí skupiny úložiště. "
"Pokus o odstranění z %(storageGroupInstanceNames)s."

#, python-format
msgid ""
"Production use of \"%(backend)s\" backend requires the Cinder controller to "
"have multipathing properly set up and the configuration option \"%(mpflag)s"
"\" to be set to \"True\"."
msgstr ""
"Použití podpůrné vrstvy \"%(backend)s\" v ostrém provozu vyžaduje, aby "
"kontrolér Cinder měl správně nastaveny vícenásobné cesty a volba nastavení "
"\"%(mpflag)s\" nastavenu na \"True\"."

#, python-format
msgid "Property %s already exists."
msgstr "Vlastnost %s již existuje."

#, python-format
msgid ""
"RBD image for backup %(backup)s of volume %(volume)s not found. Deleting "
"backup metadata."
msgstr ""
"Obraz RBD zálohy %(backup)s svazku %(volume)s nenalezen. Mazání popisných "
"dat zálohy."

#, python-format
msgid "Reconnect attempt %(attempt)s failed. Next try in %(backoff).2fs."
msgstr ""
"Pokus o znovu připojení %(attempt)s selhal. Další pokus za %(backoff).2fs."

#, python-format
msgid "Registration of image volume URI %(uri)s to image %(image_id)s failed."
msgstr "Registrace URI svazku obrazu %(uri)s k obrazu %(image_id)s selhala."

#, python-format
msgid ""
"Remaining capacity %(remainingCapacityGb)s GBs is determined from SRP pool "
"capacity and not the SLO capacity. Performance may not be what you expect."
msgstr ""
"Zbývající místo %(remainingCapacityGb)s GB je odvozeno od schopnosti zásoby "
"SRP a ne SLO. Výkon nemusí být takový, jaký jste očekávali."

#, python-format
msgid "Rename failure in cleanup of cDOT QOS policy group %(name)s: %(ex)s"
msgstr ""
"Selhání přejmenování při čistění skupiny zásady QOS cDOT %(name)s: %(ex)s"

#, python-format
msgid ""
"Report interval must be less than service down time. Current config "
"service_down_time: %(service_down_time)s, report_interval for this: service "
"is: %(report_interval)s. Setting global service_down_time to: "
"%(new_down_time)s"
msgstr ""
"Interval hlášení musí být menší než doba nečinnosti. Současné nastavení doby "
"nečinnosti služby: %(service_down_time)s, interval hlášení pro tuto službu "
"je %(report_interval)s. Nastavování doby nečinnosti globální služby na: "
"%(new_down_time)s"

msgid "Requested image is not accessible by current Tenant."
msgstr "Požadovaný obraz není dostupný současnému nájemníku."

msgid "Returning as clean tmp vol job already running."
msgstr "Probíhá vrácení protože úkol čištění dočasného svazku již probíhá."

#, python-format
msgid ""
"ScaleIO only supports volumes with a granularity of 8 GBs. The new volume "
"size is: %d."
msgstr ""
"ScaleIO podporuje pouze svazky s rozdělením po 8GB. Nová velikost svazku je: "
"%d."

#, python-format
msgid ""
"Share %s ignored due to invalid format. Must be of form address:/export."
msgstr ""
"Sdílení %s ignorováno kvůli neplatnému formátu. Musí být ve formátu adresa:/"
"export."

msgid "Silent failure of target removal detected, retry...."
msgstr "Zjištěno tiché selhání odstranění cíle, bude provede nový pokus..."

#, python-format
msgid "Snapshot \"%s\" not found."
msgstr "Snímek \"%s\" nenalezen."

#, python-format
msgid "Snapshot %(name)s already exists. Message: %(msg)s"
msgstr "Snímek %(name)s již existuje. Zpráva: %(msg)s"

#, python-format
msgid "Snapshot %s already deleted."
msgstr "Snímek %s již je smazán."

#, python-format
msgid "Snapshot still %(status)s Cannot delete snapshot."
msgstr "Snímek je stále ve stavu %(status)s. Nelze ho smazat."

#, python-format
msgid "Storage group %(name)s already exists. Message: %(msg)s"
msgstr "Skupina úložiště %(name)s již existuje. Zpráva: %(msg)s"

#, python-format
msgid "Storage sync name not found for target %(target)s on %(storageSystem)s."
msgstr ""
"Název synchronizace úložiště nenalezen v cíli %(target)s na "
"%(storageSystem)s."

msgid "Storage-assisted migration failed during retype."
msgstr "Přesun za pomocí úložiště selhal během přetypování."

#, python-format
msgid ""
"The \"netapp_size_multiplier\" configuration option is deprecated and will "
"be removed in the Mitaka release. Please set \"reserved_percentage = %d\" "
"instead."
msgstr ""
"Volba nastavení  \"netapp_size_multiplier\" je zastaralá a bude odstraněna "
"ve verzi Mitaka. Místo toho prosím nastavte \"reserved_percentage = %d\"."

msgid "The 'hplh' prefix is deprecated. Use 'hpelh' instead."
msgstr "Předpona 'hplh' je zastaralá. Místo toho použijte 'hpelh'."

msgid "The MCS Channel is grouped."
msgstr "Kanál MCS je seskupen."

msgid ""
"The NAS file operations will be run as root, allowing root level access at "
"the storage backend."
msgstr ""
"Operace se souborem NAS budou spouštěny pod účtem root, umožňující přístup k "
"podpůrné vrstvě úložiště na úrovni správce."

#, python-format
msgid ""
"The NAS file operations will be run as root: allowing root level access at "
"the storage backend. This is considered an insecure NAS environment. Please "
"see %s for information on a secure NAS configuration."
msgstr ""
"Operace se souborem NAS budou spouštěny pod účtem root: umožňující přístup k "
"podpůrné vrstvě úložiště na úrovni správce. Toto je považováno za nebezpečné "
"prostředí NAS. Pro další informace o bezpečném nastavení NFS si přečtěte %s."

msgid ""
"The NAS file permissions mode will be 666 (allowing other/world read & write "
"access)."
msgstr ""
"Režim oprávnění souboru NAS bude 666 (umožňující ostatním/světu přístup ke "
"čtení a zápisu)."

#, python-format
msgid ""
"The NAS file permissions mode will be 666 (allowing other/world read & write "
"access). This is considered an insecure NAS environment. Please see %s for "
"information on a secure NFS configuration."
msgstr ""
"Režim oprávnění souboru NAS bude 666 (umožňující ostatním/světu přístup ke "
"čtení a zápisu). Toto je považováno za nebezpečné prostředí NAS. Pro další "
"informace o bezpečném nastavení NFS si přečtěte %s."

msgid ""
"The VMAX plugin only supports Retype. If a pool based migration is necessary "
"this will happen on a Retype From the command line: cinder --os-volume-api-"
"version 2 retype <volumeId> <volumeType> --migration-policy on-demand"
msgstr ""
"Zásuvný modul VMAX podporuje pouze přetypování. Pokud je přesun pomocí "
"zásoby nutný, bude proveden při přetypování z příkazové řádky: cinder --os-"
"volume-api-version 2 retype <volumeId> <volumeType> --migration-policy on-"
"demand"

#, python-format
msgid ""
"The colon in vendor name was replaced by underscore. Updated vendor name is "
"%(name)s\"."
msgstr ""
"Pomlčka v názvu prodejce byla nahrazena podtržítkem. Aktualizovaný název "
"prodejce je %(name)s\"."

#, python-format
msgid "The device %s won't be cleared."
msgstr "Zařízení %s nebude vyčištěno."

msgid ""
"The option 'netapp_storage_pools' is deprecated and will be removed in the "
"future releases. Please use the option 'netapp_pool_name_search_pattern' "
"instead."
msgstr ""
"Volba 'netapp_storage_pools' je zastaralá a bude odstraněna v budoucích "
"verzích. Místo toho prosím použijte 'netapp_pool_name_search_pattern'."

msgid ""
"The option 'netapp_volume_list' is deprecated and will be removed in the "
"future releases. Please use the option 'netapp_pool_name_search_pattern' "
"instead."
msgstr ""
"Volba 'netapp_volume_list' je zastaralá a bude odstraněna v budoucích "
"verzích. Místo toho prosím použijte 'netapp_pool_name_search_pattern'."

msgid ""
"The primary array is not reachable at this time. Since replication is "
"enabled, listing replication targets and failing over a volume can still be "
"performed."
msgstr ""
"Hlavní pole není v současnosti dostupné. Protože je replikace povolená může "
"být stále proveden výpis cílů replikace a zavedení záložního systému."

#, python-format
msgid "The provisioning: %(provisioning)s is not valid."
msgstr "Poskytování %(provisioning)s není platné."

#, python-format
msgid ""
"The volume: %(volumename)s was not first part of the default storage group "
"for FAST policy %(fastPolicyName)s."
msgstr ""
"Svazek %(volumename)s nebyl první částí výchozí skupiny úložiště pro zásadu "
"FAST %(fastPolicyName)s."

#, python-format
msgid ""
"The volume: %(volumename)s. was not first part of the default storage group "
"for FAST policy %(fastPolicyName)s."
msgstr ""
"Svazek %(volumename)s nebyl první částí výchozí skupiny úložiště pro zásadu "
"FAST %(fastPolicyName)s."

#, python-format
msgid ""
"There are no datastores matching new requirements; can't retype volume: %s."
msgstr ""
"Neexistují žádná datová úložiště odpovídající novým požadavkům; nelze "
"přetypovat svazek %s."

#, python-format
msgid "Trying to boot from an empty volume: %s."
msgstr "Pokus o zavedení z prázdného svazku: %s."

#, python-format
msgid ""
"Unable to clone image_volume for image %(image_id)s will not create cache "
"entry."
msgstr ""
"Nelze klonovat svazek obrazu %(image_id)s, položka v mezipaměti nebude "
"vytvořena."

#, python-format
msgid "Unable to create folder %s"
msgstr "Nelze vytvořit složku %s"

#, python-format
msgid "Unable to create snapshot %s"
msgstr "Nelze vytvořit snímek %s"

#, python-format
msgid "Unable to delete Protection Group Snapshot: %s"
msgstr "Nelze smazat snímek ochranné skupiny: %s"

#, python-format
msgid "Unable to delete Protection Group: %s"
msgstr "Nelze smazat ochrannou skupinu: %s"

#, python-format
msgid "Unable to delete space %(space)s"
msgstr "Nelze smazat prostor %(space)s"

#, python-format
msgid ""
"Unable to ensure space for image-volume in cache. Will skip creating entry "
"for image %(image)s on host %(host)s."
msgstr ""
"Nelze zajistit prostor v mezipaměti pro obraz-svazek. Bude přeskočeno "
"vytváření záznamu obrazu %(image)s na hostiteli %(host)s."

#, python-format
msgid ""
"Unable to extend volume: %(vol)s to size: %(size)s on current datastore due "
"to insufficient space."
msgstr ""
"Nelze rozšířit svazek: %(vol)s na velikost %(size)s GB na současném datovém "
"úložišti z důvodu nedostatku místa."

#, python-format
msgid "Unable to find Masking view: %(view)s."
msgstr "Nelze najít maskování: %(view)s."

#, python-format
msgid "Unable to find snapshot %s"
msgstr "Nelze najít snímek %s"

msgid ""
"Unable to get internal tenant context: Missing required config parameters."
msgstr ""
"Nelze získat vnitřní kontext nájemníka: Chybí požadované parametry nastavení."

msgid "Unable to get rados pool stats."
msgstr "Nelze získat statistiky zásoby rados."

msgid "Unable to get storage tiers from tier policy rule."
msgstr "Nelze získat vrstvy úložiště z pravidla zásady vrstvy."

#, python-format
msgid "Unable to locate volume:%s"
msgstr "Nelze nalézt svazek: %s"

msgid "Unable to poll cluster free space."
msgstr "Nelze se dotázat na prostor bez clusteru."

#, python-format
msgid "Unable to rename %(old_name)s, error message: %(error)s"
msgstr "Nelze přejmenovat %(old_name)s, chybová zpráva: %(error)s"

#, python-format
msgid "Unable to update host type for host with label %(l)s. %(e)s"
msgstr "Nelze aktualizovat typ hostitele se jmenovkou %(l)s. %(e)s"

#, python-format
msgid "Unable to update stats on non-initialized Volume Group: %s"
msgstr "Nelze aktualizovat statistiky v nezavedené skupině svazku: %s"

#, python-format
msgid "Unexpected exception during image cloning in share %s"
msgstr "Neočekávaná výjimka při klonování obrazu ve sdílení %s"

msgid "Unexpected exception while listing used share."
msgstr "Neočekávaná výjimka při vypisování použitých sdílení."

msgid "Unexpected exception while short listing used share."
msgstr ""
"Při zkráceném vypisování použitého sdílení došlo k neočekávané výjimce."

#, python-format
msgid ""
"Unknown error occurred while checking mount point: %s Trying to continue."
msgstr ""
"Při kontrole bodu připojení nastala neznámá chyba: %s. Probíhá pokus o "
"pokračování."

#, python-format
msgid "Update driver status failed: %(config_group)s is uninitialized."
msgstr "Aktualizace stavu ovladače selhala: %(config_group)s není zavedeno."

msgid "Use of empty request context is deprecated"
msgstr "Použití kontextu prázdné žádosti je zastaralé"

#, python-format
msgid ""
"Vendor unique property \"%(property)s\" must start with vendor prefix with "
"colon \"%(prefix)s\". The property was not registered on capabilities list."
msgstr ""
"Vlastnost jedinečná prodejci \"%(property)s\" musí začínat předponou "
"prodejce \"%(prefix)s\" s pomlčkou. Vlastnost nebyla registrována v seznamu "
"schopností."

msgid "Verify certificate is not set, using default of False."
msgstr "Ověření certifikátu nenastaveno, použito výchozí nastavení: False."

#, python-format
msgid "Virtual Volume Set '%s' doesn't exist on array."
msgstr "Sada virtuálního svazku '%s' neexistuje v poli."

#, python-format
msgid "Volume \"%s\" not found."
msgstr "Svazek \"%s\" nenalezen."

#, python-format
msgid "Volume %(name)s already presented (%(status)d)!"
msgstr "Svazek %(name)s již je přítomen na (%(status)d)!"

#, python-format
msgid "Volume %(volume)s is not in any masking view."
msgstr "Svazek %(volume)s není v žádném maskování."

#, python-format
msgid ""
"Volume %(volumeName)s was not first part of the default storage group for "
"the FAST Policy."
msgstr ""
"Svazek %(volumeName)s nebyl první částí výchozí skupiny úložiště pro zásadu "
"FAST."

#, python-format
msgid "Volume %(volume_id)s already deleted."
msgstr "Svazek %(volume_id)s již byl smazán."

#, python-format
msgid "Volume %(volume_id)s cannot be retyped because it has snapshot."
msgstr "Svazek %(volume_id)s nemůže být přetypován protože má snímek."

#, python-format
msgid "Volume %(volume_id)s cannot be retyped during attachment."
msgstr "Svazek %(volume_id)s nemůže být během připojování přetypován."

#, python-format
msgid "Volume %s already deleted."
msgstr "Svazek %s již je smazán."

#, python-format
msgid "Volume %s does not exist."
msgstr "Svazek %s neexistuje."

#, python-format
msgid "Volume %s does not have provider_location specified, skipping"
msgstr "Svazek %s nemá zadáno umístění poskytovatele, je přeskočen"

#, python-format
msgid "Volume %s does not have provider_location specified, skipping."
msgstr "Svazek %s nemá zadáno umístění poskytovatele, je přeskočen."

#, python-format
msgid "Volume %s is not found!, it may have been deleted."
msgstr "Svazek %s nenalezen! Mohl být smazán."

#, python-format
msgid "Volume %s was not found while trying to delete it."
msgstr "Svazek %s nebyla nalezen při pokusu o jeho smazání."

#, python-format
msgid ""
"Volume : %(volumeName)s is not currently belonging to any storage group."
msgstr "Svazek: %(volumeName)s v současnosti nepatří k žádné skupině úložiště."

#, python-format
msgid "Volume copy job for src vol %s not found."
msgstr "Úkol kopírování svazku pro zdrojový svazek %s nenalezen."

#, python-format
msgid "Volume deletion failed with message: %s"
msgstr "Smazání svazku selhalo se zprávou: %s"

#, python-format
msgid "Volume is not writable. Please broaden the file permissions. Mount: %s"
msgstr ""
"Nelze zapisovat na svazek. Prosím rozšiřte oprávnění souboru. Připojení: %s"

#, python-format
msgid "Volume not found. %s"
msgstr "Svazek nenalezen. %s"

#, python-format
msgid "Volume path %s does not exist, nothing to remove."
msgstr "Cesta svazku %s neexistuje, není nic k odstranění."

msgid "Volume refresh job already running. Returning..."
msgstr "Úkol obnovy svazku již běží. Vrácení..."

#, python-format
msgid "Volume still %(status)s Cannot delete volume."
msgstr "Svazek je stále ve stavu %(status)s. Nelze ho smazat."

msgid "Volume type will be changed to be the same as the source volume."
msgstr "Typ svazku bude změněn aby odpovídal typu zdrojového svazku"

#, python-format
msgid ""
"Volume: %(volumeName)s Does not belong to storage group %(defaultSgName)s."
msgstr "Svazek: %(volumeName)s nepatří do skupiny úložiště %(defaultSgName)s."

#, python-format
msgid "Volume: %(volumeName)s is not currently belonging to any storage group."
msgstr "Svazek %(volumeName)s v současnosti nepatří k žádné skupině úložiště."

#, python-format
msgid "Volume: %s is in use, can't retype."
msgstr "Svazek %s se používá, nelze přetypovat."

#, python-format
msgid "_get_vdisk_map_properties: Did not find a preferred node for vdisk %s."
msgstr ""
"Získání vlastnosti mapy virtuálního disku: Nenalezen upřednostňovaný uzel "
"pro virtuální disk %s."

#, python-format
msgid "_migrate_cleanup on : %(volumeName)s."
msgstr "Vyčištění přesunu pro %(volumeName)s."

#, python-format
msgid "_migrate_rollback on : %(volumeName)s."
msgstr "Zpětné vrácení přesunu pro %(volumeName)s."

msgid "_remove_device: invalid properties or device."
msgstr "Odstranění zařízení: Neplatné vlastnosti nebo zařízení."

#, python-format
msgid ""
"_unmap_vdisk_from_host: Multiple mappings of volume %(vdisk_name)s found, no "
"host specified."
msgstr ""
"Odmapování virtuálního disku od hostitele: Nalezeno vícero mapování svazku "
"%(vdisk_name)s, nezadán žádný hostitel."

#, python-format
msgid ""
"_unmap_vdisk_from_host: No mapping of volume %(vol_name)s to any host found."
msgstr ""
"Odmapování virtuálního disku od hostitele: Nenalezeno žádné mapování svazku "
"%(vol_name)s k žádnému hostiteli."

msgid "flush() not supported in this version of librbd"
msgstr "flush() není podporován touto verzí librbd"

#, python-format
msgid ""
"iSCSI IP: '%s' was not found in hpe3par_iscsi_ips list defined in cinder."
"conf."
msgstr ""
"iSCSI IP: '%s' nebylo nalezeno v seznamu hpe3par_iscsi_ips zadaném v cinder."
"conf."

#, python-format
msgid "initialize_connection: Did not find a preferred node for volume %s."
msgstr "Zavedení spojení: Nenalezen upřednostňovaný uzel pro svazek %s."

#, python-format
msgid "ldev(%(ldev)d) is already mapped (hlun: %(hlu)d)"
msgstr "ldev(%(ldev)d) již je mapováno (hlun: %(hlu)d)"

#, python-format
msgid "object %(key)s of type %(typ)s not found, %(err_msg)s"
msgstr "Objekt %(key)s typu %(typ)s nenalezen, %(err_msg)s"

msgid "qemu-img is not installed."
msgstr "qemu-img není nainstalováno."

#, python-format
msgid "snapshot: %s not found, skipping delete operation"
msgstr "snímek: %s nenalezeno, operace mazání je přeskočena"

#, python-format
msgid "snapshot: %s not found, skipping delete operations"
msgstr "snímek: %s nenalezeno, operace mazání jsou přeskočeny"

#, python-format
msgid ""
"srstatld requires WSAPI version '%(srstatld_version)s' version '%(version)s' "
"is installed."
msgstr ""
"srstatld vyžaduje verzi WSAPI '%(srstatld_version)s'. Naisntalovaná verze je "
"'%(version)s'."

msgid "terminate_connection: lun map not found"
msgstr "Ukončení připojení: Mapa lun nenalezena"

#, python-format
msgid ""
"unmap_vol_from_host: Multiple mappings of volume %(vol_name)s found, no host "
"specified."
msgstr ""
"Odmapování svazku od hostitele: Nalezeno vícero mapování svazku "
"%(vol_name)s, nezadán žádný hostitel."

#, python-format
msgid ""
"unmap_vol_from_host: No mapping of volume %(vol_name)s to any host found."
msgstr ""
"Odmapování svazku od hostitele: Nenalezeno žádné mapování svazku "
"%(vol_name)s k žádnému hostiteli."

#, python-format
msgid ""
"unmap_vol_from_host: No mapping of volume %(vol_name)s to host %(host)s "
"found."
msgstr ""
"Odmapování svazku od hostitele: Nenalezeno žádné mapování svazku "
"%(vol_name)s k hostiteli %(host)s."

#, python-format
msgid "volume service is down. (host: %s)"
msgstr "Služba svazku je mimo provoz. (Hostitel: %s)"

msgid "volume_tmp_dir is now deprecated, please use image_conversion_dir."
msgstr ""
"volume_tmp_dir je nyní zastaralé, prosím použijte image_conversion_dir."

#, python-format
msgid "warning: Tried to delete vdisk %s but it does not exist."
msgstr "Varování: Pokus o smazání virtuálního disku %s, který ale neexistuje."

#, python-format
msgid ""
"zfssa_initiator: %(ini)s wont be used on zfssa_initiator_group= %(inigrp)s."
msgstr ""
"Zavaděč zfssa: %(ini)s nebude použit ve skupině zavaděče zfssa= %(inigrp)s."

msgid ""
"zfssa_initiator_config not found. Using deprecated configuration options."
msgstr "zfssa_initiator_config nenalezeno. Použity zastaralé volby nastavení."
